storage/src/test/java/org/apache/uniffle/storage/handler/impl/LocalFileServerReadHandlerTest.java

```diff
@@ -80,7 +78,7 @@ public void testDataInconsistent() throws Exception {
     int readBufferSize = 13;
     int bytesPerSegment = ((readBufferSize / blockSize) + 1) * blockSize;
     List<byte[]> segments = LocalFileHandlerTestBase.calcSegmentBytes(expectedData,
-        bytesPerSegment, actualWriteDataBlock);
+        bytesPerSegment, blockIds.subList(0, actualWriteDataBlock));
```

Comment:
@zuston because we need to get subList here.

Reply:
The size of expectedBlockIds is expected in this method of `calcSegmentBytes` invoking. Right?